lib/rucio/api/did.py

```diff
@@ -359,6 +361,26 @@ def list_content_history(scope, name, vo='def', session=None):
         yield api_update_return_dict(d, session=session)
 
 
+@stream_session
+def bulk_list_files(dids: "DIDListType", long: bool = False, vo: str = 'def', session: "Optional[Session]" = None) -> "Iterator[DIDType]":
+    """
+    List file contents of a list of data identifier.
+
+    :param dids:       A list of dids.
+    :param long:       A boolean to choose if more metadata are returned or not.
+    :param vo:         The VO to act on.
+    :param session:    The database session in use.
+    """
+
+    for did_ in dids:
```

Comment:
extra underscore?  (also in the other file)?

Reply:
Done on purpose otherwise will get trouble with this import : 
```
from rucio.core import did, naming_convention, meta as meta_core
``